.github/scripts/verify-synopsis.pl

```diff
@@ -26,39 +26,41 @@
 my @files = @ARGV;
 my $cfile = "test.c";
 
-if($files[0] eq "-h") {
+if ($files[0] eq "-h") {
     print "Usage: verify-synopsis [man pages]\n";
     exit;
 }
 
 sub testcompile {
-    my $rc = system("gcc -c test.c -DCURL_DISABLE_TYPECHECK -DCURL_ALLOW_OLD_MULTI_SOCKET -I include") >> 8;
+    my $rc = system(
+"gcc -c test.c -DCURL_DISABLE_TYPECHECK -DCURL_ALLOW_OLD_MULTI_SOCKET -I include"
+    ) >> 8;
     return $rc;
 }
 
-
 sub extract {
-    my($f) = @_;
-    my $syn = 0;
-    my $l = 0;
+    my ($f)   = @_;
+    my $syn   = 0;
+    my $l     = 0;
```

Comment:
Another case of vertical alignment that is difficult to maintain manually.
If there is a way to disable, I'd be in favor.